### ISSUE TITLE
position of top level admin nav for plugins

### ISSUE THREAD

**Issue by JahlomP** [pencilblue/pencilblue#416]
When you create a top level admin nav for a plugin, it comes last. I think it should come before the settings nav for it to feel more natural.

**Comment by blakecallens** [comment]
I agree with this. @brianhyder?

**Comment by brianhyder** [comment]
I agree. The admin navigation should be a weighted system where each
element gets a weight and sorted accordingly. It's just not something that
is high on our priority list.

**Comment by JahlomP** [comment]
ok
